repositories.bzl

```diff
@@ -1,22 +1,295 @@
+        "mixer/v1/config/descriptor/monitored_resource_descriptor.proto": "istio.io/api/mixer/v1/config/descriptor",
+        "mixer/v1/config/descriptor/principal_descriptor.proto": "istio.io/api/mixer/v1/config/descriptor",
+        "mixer/v1/config/descriptor/quota_descriptor.proto": "istio.io/api/mixer/v1/config/descriptor",
+        "mixer/v1/config/descriptor/value_type.proto": "istio.io/api/mixer/v1/config/descriptor",
+    },
+    imports = [
+        "../../external/com_github_google_protobuf/src",
+    ],
+    inputs = DESCRIPTOR_FILE_GROUP + [
+        "@com_github_google_protobuf//:well_known_protos",
+    ],
+    protos = [
+        "mixer/v1/config/cfg.proto",
+    ],
+    verbose = 0,
+    visibility = ["//visibility:public"],
+    with_grpc = False,
+)
+
+gogoslick_proto_library(
+    name = "mixer/v1/config/descriptor",
+    importmap = {
+        "google/protobuf/duration.proto": "github.com/gogo/protobuf/types",
+    },
+    imports = [
+        "../../external/com_github_google_protobuf/src",
+    ],
+    inputs = [
+        "@com_github_google_protobuf//:well_known_protos",
+    ],
+    protos = DESCRIPTOR_FILE_GROUP,
+    verbose = 0,
+    visibility = ["//visibility:public"],
+    with_grpc = False,
+    deps = [
+        "@com_github_gogo_protobuf//sortkeys:go_default_library",
+        "@com_github_gogo_protobuf//types:go_default_library",
+    ],
+)
+
+filegroup(
+    name = "mixer/v1/config/descriptor_protos",
+    srcs = DESCRIPTOR_FILE_GROUP,
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "mixer/v1/config_fixed",
+    srcs = [":mixer/v1/config_gen"],
+    outs = ["fixed_cfg.pb.go"],
+    cmd = "sed " +
+          "-e 's/*google_protobuf.Struct/interface{}/g' " +
+          "-e 's/ValueType_VALUE_TYPE_UNSPECIFIED/VALUE_TYPE_UNSPECIFIED/g' " +
+          "$(location :mixer/v1/config_gen) | $(location @org_golang_x_tools_imports//:goimports) > $@",
+    message = "Applying overrides to cfg proto",
+    tools = ["@org_golang_x_tools_imports//:goimports"],
+)
+
+filegroup(
+    name = "mixer/v1/attributes_file",
+    srcs = ["mixer/v1/global_dictionary.yaml"],
+    visibility = ["//visibility:public"],
+)
+"""
     if use_local:
         native.new_local_repository(
-            name = name,
-            build_file = build_file,
-            path = path
+            name = "com_github_istio_api",
+            build_file_content = ISTIO_API_BUILD_FILE,
+            path = "../api",
         )
     else:
-        native.new_git_repository(
-            name = name,
-            build_file = build_file,
-	    commit = commit,
-	    remote = remote
-        )
+      native.new_git_repository(
+          name = "com_github_istio_api",
+          build_file_content = ISTIO_API_BUILD_FILE,
+          commit = ISTIO_API_SHA,
+          remote = "https://github.com/istio/api.git",
+      )
+
+def go_googleapis_repositories():
```

Comment:
same question here: can we make this `googleapis.bzl` that gets loaded here (but can be used elsewhere)?